recipe/conda_build_config.yaml

```diff
@@ -179,18 +185,18 @@ pyqt:
 qt:
   - 5.6
 readline:
-  - 7
+  - 7.0
 r_base:
   - 3.3.2
   - 3.4.1
 scipy:
   - 0.19
 snappy:
-  - 1
+  - 1.1.6
 sox:
-  - 14.4
+  - 14.4.2
```

Comment:
`sox` is [really unstable]( https://abi-laboratory.pro/tracker/timeline/sox/ ) even between patch versions. So this needs to be tight. Again same as [what we had]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L121 ).